src/JSDOMDomProvider.js

```diff
@@ -46,7 +46,7 @@ export default class JSDOMDomProvider {
               availHeight: { value: height },
             });
             win.requestAnimationFrame = (callback) => setTimeout(callback, 0);
-            win.cancelAnimationFrame = () => {};
+            win.cancelAnimationFrame = clearTimeout;
```

Comment:
any reason not to use a real raf shim here, like https://npmjs.com/raf ?

Reply:
Or on that note, why not bring in airbnb-browser-shims or some similar package?

https://github.com/airbnb/browser-shims

Reply:
I’ll have a look at this separately 👍